src/map.h

```diff
@@ -972,6 +972,12 @@ class map
         void remove_trap( const tripoint &p );
         const std::vector<tripoint> &trap_locations( trap_id t ) const;
 
+		/*
+		 * Spawns byproducts from items destroyed in fire.
+		 * Byproducts are declared in JSON as:
+		 * "burn_products": [ [ "X", float efficiency ], [ "Y", float efficiency ] ]
```

Comment:
This information is quite useful, but it does not belong here. People that want to edit JSON files won't search here. And when the format is actually changed, the loading code in "material.cpp" is updated, but most likely not the comment on this function. It might be more suitable as comment for `material_type::_burn_products`.